### ISSUE TITLE
What happened to the audio recognition feature?

### ISSUE THREAD

**Issue by brooke-heaton** [aschober/vinyl-cast#7]
Just curious. I saw that you struck that out of the ReadMe

**Comment by aschober** [comment]
Yes, removed the Audio ACR piece as looks like Gracenote no longer gives access to their SDK to the public and haven't found any other easily useable providers that are free / super-low-cost.

**Comment by WillGeller** [comment]
@aschober Totally love what you have created with vinyl-cast and just got a google mini speaker so I can start using it. I do wish the the Audio meta data feed feature was still somehow available, I would be willing to do some research and see if there is any APIs that could replace Gracenote and let you know if anything looks promising. I am not a Java expert but could probably follow the code history to see how it was done before the feature was removed.

**Comment by WillGeller** [comment]
It does seem that Gracenote has a community license for developers and non commercial use that may be suited to this open source project https://developer.gracenote.com/en/frequently-asked-questions, Is the code for connecting with GNSDK still in the git history of this repo?

**Comment by aschober** [comment]
@WillGeller appreciate the shout out.

I looked at Gracenote SDK again last month and while they still have the developer site up and I was able to login with my old Gracenote account, I was not able to get the SDK to initialize using my SDK access key. It seems like it's a zombie but I have not tried reaching out to their sales/support team to ask.

I am exploring other Audio ACR providers to add the feature back in, but I will likely have to make it a "paid" feature as there are pretty limited options for free/trial usage. 

Open to any research you find though.
